tools/mfassignr/mfassignr_histnoise.xml

```diff
@@ -0,0 +1,44 @@
+<tool id="mfassignr_histnoise" name="MFAssignR HistNoise" version="@TOOL_VERSION@+galaxy0">
+    <description>Noise level assessment using the HistNoise.</description>
+    <macros>
+        <import>macros.xml</import>
+        <import>help.xml</import>
+    </macros>
+    <edam_topics>
+        <edam_topic>topic_3172</edam_topic>    
+    </edam_topics>
+    <expand macro="creator" />
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript -e 'source("${run_script}")'
```

Comment:
Would something like that work as well? https://github.com/galaxyproject/tools-iuc/blob/6da96d865a3a557cfa3ad09e1cfa830519e73748/tools/ruvseq/ruvseq.xml#L46

So without source()

Reply:
Yes.